light-zk.js/tests/utxo.test.ts

```diff
@@ -352,10 +384,49 @@ describe("Utxo Functional", () => {
       verifierProgramLookupTable:
         lightProvider.lookUpTables.verifierProgramLookupTable,
     });
-    if (receivingUtxo1) {
-      Utxo.equal(poseidon, receivingUtxo, receivingUtxo1, true);
+    if (receivingUtxo1Unchecked) {
+      Utxo.equal(poseidon, receivingUtxo, receivingUtxo1Unchecked, true);
     } else {
-      throw new Error("decrypt failed");
+      throw new Error("decrypt unchecked failed");
+    }
+
+    const receivingUtxoNoAes = new Utxo({
+      poseidon,
+      assets: inputs.assets,
+      amounts: inputs.amounts,
+      account: new Account({ poseidon }),
+      blinding: inputs.blinding,
+      index: inputs.index,
+      assetLookupTable: lightProvider.lookUpTables.assetLookupTable,
+      verifierProgramLookupTable:
+        lightProvider.lookUpTables.verifierProgramLookupTable,
+    });
+
+    // encrypt
+    const encBytesNaclNoAes = await receivingUtxoNoAes.encrypt(
+      poseidon,
+      MerkleTreeConfig.getTransactionMerkleTreePda(),
+    );
+
+    // decrypt
+    const receivingUtxo1NoAes = await Utxo.decrypt({
+      poseidon,
+      encBytes: encBytesNaclNoAes,
+      account: inputs.keypair,
+      index: inputs.index,
+      merkleTreePdaPublicKey: MerkleTreeConfig.getTransactionMerkleTreePda(),
+      aes: false,
+      commitment: new anchor.BN(
+        receivingUtxoNoAes.getCommitment(poseidon),
+      ).toBuffer("le", 32),
+      assetLookupTable: lightProvider.lookUpTables.assetLookupTable,
+      verifierProgramLookupTable:
+        lightProvider.lookUpTables.verifierProgramLookupTable,
+    });
+    if (typeof receivingUtxo1NoAes == "boolean") {
+      assert.equal(receivingUtxo1NoAes, false);
```

Comment:
this is because of this -> https://github.com/Lightprotocol/light-protocol/blob/92c986106169a6dcccf460d3d3fcef0962d9292f/light-zk.js/src/utxo.ts?plain=1#L842-L845

Reply:
and this -> 
https://github.com/Lightprotocol/light-protocol/blob/92c986106169a6dcccf460d3d3fcef0962d9292f/light-zk.js/src/utxo.ts?plain=1#L826

Reply:
so in that case we'll have:
a) no UTXO found
b) no collision

so it returns FALSE